assets/js/googlesitekit/widgets/components/Widget.js

```diff
@@ -22,21 +22,34 @@
 import classnames from 'classnames';
 import { node, string } from 'prop-types';
 
-const Widget = ( { children, slug } ) => {
+/**
+ * Internal dependencies
+ */
+import { WIDGET_WIDTHS } from '../datastore/constants';
+
+function Widget( { children, slug, width } ) {
+	const classes = classnames( `googlesitekit-widget googlesitekit-widget--${ slug }`, {
```

Comment:
Why move this up here? It's just that we tend to place `classnames` calls directly in the component's `className` attribute elsewhere, as it helps directly connect the classnames to the component they belong to. In this instance it's straightforward, but elsewhere doing this would make it more difficult to connect the classnames to a component.

Reply:
Updated. Moved directly to the className attribute.